renderer/html_test.go

```diff
@@ -196,13 +196,6 @@ func TestRenderHTML_Footer(t *testing.T) {
 
```

Comment:
I think we should look to remove this entire test (i.e. from the convey) as it seems to only care about the back to table links we have removed.